Resources/android/titanium-mapbox/index.js

```diff
@@ -0,0 +1,9 @@
+import MapView from './mapview';
+import Annotation from './annotation';
+
+const Mapbox = require('com.mapbox.mapboxsdk.Mapbox');
+const Activity = require('android.app.Activity');
+
+Mapbox.getInstance(new Activity(Ti.Android.currentActivity), 'YOUR_MAPBOX_ACCESS_TOKEN');
```

Comment:
I'm not sure if this will work correctly here, since it may have to run delayed for the current activity stuff to work. That's why I did it lazily inside the createView call.

Reply:
I was hoping it is executed once anything is imported, which seems to work okay. The problem is that there are mapbox API's that do not require a map-view, like geocoders. Maybe do an `initialize` method? iOS can actually also be configured by code, so I'd remove the plist-key and use one central method.

Reply:
Doing this one:
```js
import MapView from './mapview';
import Annotation from './annotation';

const MGLAccountManager = require('Mapbox/MGLAccountManager');

class Mapbox {
	static initialize(apiKey) {
		if (!apiKey) { throw 'No API-key provided!'; }
		MGLAccountManager.setAccessToken(apiKey);
	}
}

export { Annotation, MapView, Mapbox }
```

Reply:
Updated